roles/postgresql/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+# tasks file for postgresql
+- name: make sure the CA certificates are available
+  become: true
+  apt:
+    pkg: ca-certificates
+    state: present
+
+- name: add postgres repository apt-key
+  become: true
+  apt_key:
+    url: "{{ postgres_apt_key_url }}"
+    state: present
+
+- name: add postgres repository
+  become: true
+  apt_repository:
+    repo: "{{ postgres_apt_repository }}"
+    update_cache: true
+    state: present
+
```

Comment:
what are these needed for?

Reply:
they're necessary to install the desired version of the postgresql client library in the next step.